onfido/live_photos.py

```diff
@@ -0,0 +1,20 @@
+"""Wrapper for the live photos endpoint."""
+import mimetypes
+
+from. api_resource import ApiResource
+
+
+class LivePhotos(ApiResource):
+    """Represents the Live Photos API endpoint."""
+    def create(self, applicant_id, live_photo, filename):
+        """POST to /live_photos
+
+        Args:
+            applicant_id: ID of the applicant this photo belongs to (string).
+            live_photo: The live photo to upload (file-like object).
+            filename: The name of the live photo file (string).
+        """
+        content_type = mimetypes.guess_type(filename)
+        file = {'file': (filename, live_photo, content_type[0])}
```

Comment:
don't use `file` as a variable name, it overwrites the the buildin file method (at least in Python 2)